cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1048,15 +1049,19 @@ private void startExecution(LoadMonitor loadMonitor,
    */
   private void sanityCheckOngoingMovement() throws OngoingExecutionException {
     boolean hasOngoingPartitionReassignments;
+    Map<TopicPartition, PartitionReassignment> ongoingPartitionReassignments;
     try {
-      hasOngoingPartitionReassignments = hasOngoingPartitionReassignments();
+       ongoingPartitionReassignments = ExecutionUtils.ongoingPartitionReassignments(_adminClient);
+      // avoid calling hasOngoingPartitionReassignments() to save 1 request.
```

Comment:
I think it's fine to remove this comment. This is probably only be useful during review time.